GOVERNANCE.rst

```diff
@@ -384,27 +414,41 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding weekends
+   since the beginning of the stage-1 "reasonable time for consideration" period,
+   the decision is made through a simple majority vote (with tie breaking) among the CC
```

Comment:
```suggestion
-  If no consensus can be found, the decision is made through a simple majority vote (with tie breaking) among the CC
```

Reply:
there may not always be a CC meeting within that period, the role of the CC meeting in that case would also be to try again to find a consensus, by perhaps taking parts of the PR and rejecting other, etc

Reply:
this would muddy the definition of what it means "no consensus can be found". When do we pass to the CC?

I really don't like that your suggested changes are reducing accountability and guarantee of speedy process. Whether you intend it or not, removal of clear condtions and clear decision making points is directly hampering effective governance.